src/Akeneo/Connectivity/Connection/back/Application/Webhook/Service/EventsApiDebugWebhookClientLogger.php

```diff
@@ -10,14 +10,27 @@
  * @copyright 2020 Akeneo SAS (http://www.akeneo.com)
  * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
  */
-interface EventsApiDebugResponseErrorLogger
+interface EventsApiDebugWebhookClientLogger
```

Comment:
```suggestion
interface EventsApiRequestLogger
```
or something like that